requirements.txt

```diff
@@ -1,3 +1,4 @@
 torch>=1.4.0
 torchvision>=0.2.2
-scikit-learn>=0.23.0
\ No newline at end of file
+scikit-learn>=0.23.0
+tensorboard
```

Comment:
We need to find the minimal version for tensorboard that is compatible with other dependencies.

Reply:
Sorry, I can't tell the minimal version of tensorboard. I've found nothing of version conflict in pytorch's docs and I've tried several version of tensorboard among v1.13 to v2.4, everything works well.

Reply:
Sorry, I can't tell the minimal version of tensorboard. I've found nothing of version conflict in pytorch's docs and I've tried several version of tensorboard among v1.13 to v2.4, everything works well.

Reply:
How about pinning the version of tensorboard to `2.*`? I have checked the PyPI, the version `2.0.0` was released in Sep, 2019, this should be appropriate for most users.